spec/features/reports_manage_spec.rb

```diff
@@ -92,5 +92,28 @@
       expect(page).to have_content(percentage_off_estimate_total_best)
       expect(page).to have_content(percentage_off_estimate_total_worst)
     end
+
+    # there's a bug that allows estimators to have more than 1 estimate for a given story
+    # even after that bug gets fixed, we have to be sure we handle old data correctly
+    context "by the same user" do
+      it "uses the first estimation" do
+        anotherEstimation =
+          FactoryBot.create(:estimate, story: story, user: user,
+            best_case_points: 8,
+            worst_case_points: 13,
+            created_at: 2.minutes.from_now)
```

Comment:
@arielj hm, It looks a bit weird to me to have a `anotherEstimation` variable here. Maybe it'd be better to move the factory to a `before` block:

```
context "by the same user" do
  before do
    FactoryBot.create(:estimate, story: story, user: user, best_case_points: 8, worst_case_points: 13, created_at: 2.minutes.from_now)
  end

  it "...
```

Reply:
personally, I don't like to have a before block for a single test and it looks weird to me xD, but I don't have a strong opinion haha, I'll move it to be a `let` like the rest of the objects that are created in that file